lib/__tests__/change-control-lambda/handler.test.ts

```diff
@@ -14,12 +10,22 @@ import * as timeWindow from '../../change-control-lambda/time-window';
 //                | |
 //                |_|
 
-jest.mock('aws-sdk');
+jest.mock('aws-sdk', () => {
+  return {
+    S3: jest.fn(() => {
+      return {
+        getObject: mockGetObject,
+      };
+    }).mockName('AWS.S3'),
+    CodePipeline: jest.fn(() => {
+      return {};
+    }).mockName('AWS.CodePipeline'),
+  };
+});
 jest.mock('../../change-control-lambda/disable-transition');
 jest.mock('../../change-control-lambda/time-window');
 
 const mockGetObject = jest.fn().mockName('AWS.S3.getObject');
-(AWS as any).S3 = jest.fn(() => ({ getObject: mockGetObject })).mockName('AWS.S3') as any;
```

Comment:
One of the dependency updates did not like `(AWS as any)` so I updated the test.